include/maliput/common/maliput_throw.h

```diff
@@ -122,6 +122,17 @@ void Throw(const char* condition, const char* func, const char* file, int line)
                                                                          __LINE__);                                 \
   } while (0)
 
+/// @def MALIPUT_VALIDATE
+/// Used to validate that a @p pred-icate passed into a function or method is
+/// true; if not, an exception with @p message is thrown.
+#define MALIPUT_VALIDATE(pred, message)                                                                            \
+  do {                                                                                                             \
+    if (!(pred)) {                                                                                                 \
+      ::maliput::common::internal::Throw<maliput::common::assertion_error>(std::string(message).c_str(), __func__, \
+                                                                           __FILE__, __LINE__);                    \
+    }                                                                                                              \
+  } while (0)
```

Comment:
What if you only have MALIPUT_VALIDATE where you pass the error type?
That way we would eliminate MALIPUT_RULES_VALIDATE which actually it isn't forbidding you to pass a non rule type error

Reply:
Done.